### ISSUE TITLE
Whales: Editable and Deleteable elements are inconsistant

### ISSUE THREAD

**Issue by upsonp** [dfo-mar-odis/dm_apps#483]
Lastly, it seems to be a bit inconsistent at the moment where we have the ability to edit data that is entered and where we don’t. For example, we cannot edit entries in the “Team Members” table or the” Recording Schedules” table (can only add or delete stages within a schedule). And, we do not have the option to delete entries made pretty much anywhere throughout the whole app. While I am doing my best to QC all of our metadata, the reality is that there are still plenty of record-keeping mistakes that aren’t discovered (or properly communicated) until well after the fact, and typos during data entry will inevitably happen. Not being able to delete entries made in error could cause things to get really messy, and my preference would be to have the ability to edit and/or delete anything that we enter. I’m not sure if there are certain places where editing data later could cause other things to “break”, but I’m happy to discuss further.

**Comment by upsonp** [comment]
I've added update buttons to both the Team Members and Recording Schedule list pages. I flipped through some of the top level entry and detail forms, but didn't really see any others that are lacking an update button at the very least.

I have no issue with adding delete buttons to some things, but we really don't want to put a delete button on a pick list driven list where deleting an element will cause integrity issues.

**Comment by upsonp** [comment]
I've added update buttons to both the Team Members and Recording Schedule list pages. I flipped through some of the top level entry and detail forms, but didn't really see any others that are lacking an update button at the very least.

I have no issue with adding delete buttons to some things, but we really don't want to put a delete button on a pick list driven list where deleting an element will cause integrity issues.
